.github/dependabot.yml

```diff
@@ -5,7 +5,6 @@ updates:
     directory: '/'
     schedule:
       interval: daily
-    open-pull-requests-limit: 10
```

Comment:
Removed those limits because they're no longer necessary (they're mostly useful when you "just enabled" dependant on a codebase with a lot of outdated dependencies). This would improve our ability to remain up-to-date...